python/GafferUI/LabelPlugValueWidget.py

```diff
@@ -176,8 +176,16 @@ def __dragEnd( self, widget, event ) :
 	def __updateDoubleClickConnection( self ) :
 
 		self.__labelDoubleClickConnection = None
+		if self.getPlug() is None :
+			return
+
+		# First try old metadata that we are phasing out.
+		renameable = Gaffer.Metadata.value( self.getPlug(), "labelPlugValueWidget:renameable" )
+		if renameable is None :
+			# Then try the official metadata we are phasing in.
+			renameable = Gaffer.Metadata.value( self.getPlug(), "renameable" )
```

Comment:
Is this backwards? If a plug has both keys for some reason, we'd want to use the value from the key we are phasing in, right?

Reply:
It is - thanks for catching that. I'd written it at a time when I thought I was going to register `"*", "renameable", False` across the board in NodeUI, and failed to update it when I didn't. Fixed in 7acd697 where I've also addressed an inconsistency in the interpretation of `Metadata.value( "readable" ) == None`.